quilt/tools/command.py

```diff
@@ -410,10 +410,13 @@ def install(package, hash=None, version=None, tag=None):
     existing_pkg = store.get_package(owner, pkg)
 
     if existing_pkg is not None:
```

Comment:
Just make it `if existing_pkg is not None and not force`. Or you could skip the `existing_pkg` lookup if `force=True`.